doc/minutes/2022-12-01_first_meeting.md

```diff
@@ -0,0 +1,73 @@
+---
+title:  "First meeting YOS Award"
+date:   "30-11-2022"
+editor_options:
+  markdown:
+    canonical: false
+---
+
+**Attendees:** <names and surnames of attendees>
+
+-   Daniel Morillo Cuadrado
+
+-   Roberto Mediavilla Torres
+
+-   Cristina Rodríguez-Prada
+
+-   Lea Francia
+
+-   Jesus Sierralaya
+
+**Scribe:** Jesus Sierralaya
+
+**Schedule:** 11:00 - 11:30 <!-- # Init time - End time -->
+
+# Action points
+
+## Project Proposals
+
+### Robert's proposal
+
+-   Cloud data management system and federated analysis
+
+-   Software Obiba Opal
+
+-   We already have a physical server
+
+### Lea's proposal
+
+-   Conducting a study from scratch to publication with GitHub
+
+### Daniel's proposal
```

Comment:
I pointed out at the meeting that I agreed with both Rober's and Lea's proposals and that a combination of the two could then be made, along Dani's direction.

Reply:
@Cristrinaranjus I think you may consider resolving this conversation now.